source/scss/_pf-basic-variables.scss

```diff
@@ -456,3 +460,33 @@ $tag-font-weight:           $pf-font-weight-bold;
 // Use a higher than normal value to ensure completely rounded edges when
 // customizing padding or font-size on labels.
 // $tag-pill-border-radius:    10rem !default;
+
+
+// Form states and alerts
+//
+// Define colors for form feedback states and, by default, alerts.
```

Comment:
Are you proposing to move the color specs of the icons error, warning, info and success from the _pf-basic-icons.scss to this file?
I'm not strong about this move but I think it works also. But not under this "alerts" section of code, since the icons are used also in other components, like cards. Anyways I'd leave this for a different PR.